### ISSUE TITLE
Do Not Include Options/Arguments Where help=argparse.SUPPRESS

### ISSUE THREAD

**Issue by derks** [kislyuk/argcomplete#113]
Currently, Argcomplete includes all options/sub-parsers/etc even if they are not visible in `--help` output (i.e. where they have `help=argparse.SUPPRESS`).  Argcomplete should not include these options/arguments/sub-parsers/etc.

**Comment by derks** [comment]
Example:

**myapp.py**

```python
#!/usr/bin/env python

import argcomplete
import argparse

parser = argparse.ArgumentParser()
parser.add_argument('-f', '--foo', help='foo option', dest='foo')
parser.add_argument('--my-hidden-option', help=argparse.SUPPRESS)
argcomplete.autocomplete(parser)
args = parser.parse_args()
```

Help Output:

```
$ ./myapp.py --help
usage: myapp.py [-h] [-f FOO]

optional arguments:
  -h, --help         show this help message and exit
  -f FOO, --foo FOO  foo option
```

Argcomplete Output:

```
$ ./myapp2.py [tab][tab]
--foo               --my-hidden-option  -h
--help              -f
```

Note that as mentioned, Argcomplete should ignore all options, positionals, sub-parsers, etc that have `help=argparse.SUPPRESS`.

**Comment by kislyuk** [comment]
Thanks, this is a great idea. Patches are welcome. This behavior should be configurable.

**Comment by kislyuk** [comment]
Released in v1.0.0, please test
